_layouts/quality_estimation.html

```diff
@@ -151,9 +158,37 @@ <h2>Language support</h2>
       </ul>
     </details>
 
-{% else %}
+  {% else %}
 
-  <p>{{ page.title }} supports an unknown number of languages.</p>
+    <p>{{ page.title }} supports an unknown number of languages.</p>
+  {% endif %}
 </section>
 
-{% endif %}
+
+{% if page.integrations.size > 0 %}
```

Comment:
I think we can remove this, because we won't include any that doesn't have any integrations, and if we did, it's fine that it's obvious that it has 0.